packages/core/src/application/application_ref.ts

```diff
@@ -716,8 +701,40 @@ export function whenStable(applicationRef: ApplicationRef): Promise<void> {
 }
 
 
+export function detectChangesInViewIfRequired(
+    isFirstPass: boolean, lView: LView, notifyErrorHandler: boolean) {
```

Comment:
Minor nit: parameter order.
```suggestion
    lView: LView, isFirstPass: boolean, notifyErrorHandler: boolean) {
```